BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/BrainCloudComms.cs

```diff
@@ -312,7 +312,8 @@ public void SetPacketTimeoutsToDefault()
             _packetTimeouts = new List<int> { 15, 20, 35, 50 };
         }
 
-        private int _authPacketTimeoutSecs = 15;
+        private List<int> _listAuthPacketTimeouts = new List<int> {0, 15, 30, 60 };
```

Comment:
So - why is 0 a value in the list? Wouldn't the initial timeout value be 16?

Reply:
(typo - meant 15 :) )

Reply:
I have no reason why 0 is in the list other than testing logs and then I forgot to take it out. I'll take it out and default it to 15.